diff --git a/git_hooks/pre-commit b/git_hooks/pre-commit
index add8c4b..807e79c 100755
--- a/git_hooks/pre-commit
+++ b/git_hooks/pre-commit
@@ -76,6 +76,16 @@ if [ "$versions_changed" -eq 1 ]; then
     git add "$versions_path"
 fi
 
+###
+### update file_version in 'hub/bbbased.lua'
+###
+
+bbbased_path='hub/bbbased.lua'
+if printf '%s\n' "$staged_files" |grep -Fxq -- "$bbbased_path"; then
+    sed -i "s/^local commit_date = .*/local commit_date = '$b36datetime'/" "$bbbased_path"
+    git add "$bbbased_path"
+fi
+
 ###
 ### return to former directory
 ###
diff --git a/hub/bbbased.lua b/hub/bbbased.lua
index 9e4bf75..c924bd4 100644
--- a/hub/bbbased.lua
+++ b/hub/bbbased.lua
@@ -1,8 +1,7 @@
 #!/usr/bin/lua
 
 -- BitBurrow base daemon
---
--- strings use single quotes unless they are user-visible English, e.g. logging
+-- note: strings use single quotes unless they are user-visible English, e.g. logging
 
 --
 -- hard-coded at time of download in get_adopt5s_script()
@@ -19,6 +18,7 @@ local log_err_route = '{log_err_route}'
 -- globals
 --
 
+local commit_date = ''
 local bbsubd = 'bb' .. subd
 local tmp_dir = os.getenv('TMPDIR')
 if tmp_dir and tmp_dir ~= '' then
